src/components/Person/Person.jsx

```diff
@@ -1 +1,20 @@
-// export const Person = ({ person }) => ();
+import './Person.scss';
+
+export const Person = ({ person }) => (
+  <section className="Person">
```

Comment:
it's a good practice to use object destructing 
```suggestion
export const Person = ({ person }) => {
  const { ... } = person;
  
  return (
    <section className="Person">
```

Reply:
Thank you very much for your comments. They are wonderful! I got it. it's very important for a person who is learning that the comments are just like yours - clear and understandable.